### DISCUSSION TITLE
Can't Access Student Developer Pack

### DISCUSSION THREAD

**Discussion by ericl3ung** [community/community#38091] [GitHub Education]
I have lost my Pro and the Student Developer Pack as I can't reverify my academic affiliation.

My Process:
> Are you a student? {I click Yes}
> You must periodically re-verify your academic affiliation to continue to qualify for GitHub Education benefits. If you are having trouble accessing your benefits. {I click submit information again}
> "You were last verified as a student on May 4, 2022. It is not necessary for you to re-verify at this time". {Message pop-up}

**Comment by samus-aran** [comment]
Hi @ericl3ung 👋 
How did you get on? Did you have to change anything to re-verify?
